src/types/Params.ts

```diff
@@ -0,0 +1,5 @@
+export interface Params {
```

Comment:
From the name, it is not entirely clear which `Params` this interface refers to, it is better to give such names so that they are clear out of context.